generators/server/templates/src/main/java/package/config/SecurityConfiguration_reactive.java.ejs

```diff
@@ -279,10 +279,12 @@ public class SecurityConfiguration {
             .pathMatchers("/api/account/reset-password/init").permitAll()
             .pathMatchers("/api/account/reset-password/finish").permitAll()
 <%_ } _%>
+<%_ if (authenticationTypeOauth2) { _%>
             .pathMatchers("/api/auth-info").permitAll()
+<%_ } _%>
             .pathMatchers("/api/admin/**").hasAuthority(AuthoritiesConstants.ADMIN)
             .pathMatchers("/api/**").authenticated()
-<%_ if (applicationTypeGateway) { _%>
+<%_ if (applicationTypeGateway && microfrontend) { _%>
             // microfrontend resources are loaded by webpack without authentication, they need to be public
             .pathMatchers("/services/*/*.js").permitAll()
             .pathMatchers("/services/*/*.js.map").permitAll()
```

Comment:
Swagger api should be exposed for non microfrontends.
```suggestion
<%_ if (applicationTypeGateway) { _%>
  <%_ if (microfrontend) { _%>
            // microfrontend resources are loaded by webpack without authentication, they need to be public
            .pathMatchers("/services/*/*.js").permitAll()
            .pathMatchers("/services/*/*.js.map").permitAll()
  <%_ } _%>
```

Reply:
I agree to some extent, but the current configurations don't point in that direction and are confusing. Current mapping protects the services `/v3/api-docs` endpoint with the ADMIN role. I initially thought of doing refactoring, but, hold onto that as we have another ticket to improve the imperative client security configurations and this can also be done along that or as a separate ticket.

Reply:
I still don't get.
`/services/*/v3/api-docs` accessibility should be the same for gateways with microfrontends and without.
`.pathMatchers("/services/*/v3/api-docs").hasAuthority(AuthoritiesConstants.ADMIN)` should be removed in this case?